tailblocks/src/App.js

```diff
@@ -1,21 +1,37 @@
-import React from 'react'
-import Header from './components/Header'
-import Hero from './components/Hero'
-import Description from './components/Description'
-import Solution from './components/Solution'
-import Technology from './components/Technology'
-import Footer from "./components/Footer"
+import React from 'react';
+import Home from './pages/home';
```

Comment:
Looks like this path doesn't exist anymore. Check the case in each import. It causes issues on Mac and Linux.

Reply:
Removed all unnecessary imports.